client/js/components/pages/logout.jsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import UserActions from '../../actions/user-actions';
+
+class Logout extends React.Component {
+
+  constructor(props) {
+    super(props);
+    UserActions.logout();
```

Comment:
@Vakrim so everytime you first render that component you trigger ```UserActions.logout()``` are you sure you want that to happend?